pkg/grpc/exec/exec.go

```diff
@@ -174,12 +186,12 @@ func (msg *MsgExecveEventUnix) RetryInternal(_ notify.Event, _ uint64) (*process
 func (msg *MsgExecveEventUnix) Retry(internal *process.ProcessInternal, ev notify.Event) error {
 	var podInfo *tetragon.Pod
 
-	proc := ev.GetProcess()
-	parent := ev.GetParent()
+	tetragonProcess := ev.GetProcess()
+	tetragonParent := ev.GetParent()
```

Comment:
this rename makes the whole change more complex, please keep proc and parent

Reply:
I've renamed them for consistency purposes, mostly because [`GetProcessExec`](https://github.com/cilium/tetragon/blob/main/pkg/grpc/exec/exec.go#L42) uses these names for the same objects. I.e `proc` for `*process.ProcessInternal` and `tetragonProcess` for `*tetragon.Process`.

In some functions process of type `*process.ProcessInternal` is named `proc`, in others - `internal`. And in [`MsgCloneEventUnix`.`Retry`](https://github.com/cilium/tetragon/blob/main/pkg/grpc/exec/exec.go#L301) `proc` has type `*tetragon.Process`. It's kind of inconvenient.

Also, i would say that [tracing.go](https://github.com/cilium/tetragon/blob/main/pkg/grpc/tracing/tracing.go) is much easier to understand, as it uses the same naming in all functions. I'll revert these changes back, but for the future i think it would be better to make variables names more consistent.